lib/pkgcloud/openstack/network/client/securityGroupRules.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * securityGroupRules.js: Instance methods for working with security group rules
+ * for Openstack Networking
+ *
+ * (C) 2015 Rackspace
+ *      Shaunak Kashyap
+ * MIT LICENSE
+ */
+
+var urlJoin = require('url-join');
+
+var securityGroupRulesResourcePath = '/security-group-rules';
+
+// Declaring variables for helper functions defined later
+var _convertSecurityGroupRuleToWireFormat;
+
+/**
+ * client.getSecurityGroupRules
+ *
+ * @description get the list of security group rules for an account
+ *
+ * @param {object|Function}   options
+ * @param {Function}          callback
+ */
+exports.getSecurityGroupRules = function (options, callback) {
```

Comment:
Correct. It has no utility at the moment but might in the future if the underlying HTTP API adds any parameters.